spec.html

```diff
@@ -21720,6 +21756,10 @@
             1. Let _result_ be the result of evaluating _module_.[[ECMAScriptCode]].
             1. Suspend _moduleCxt_ and remove it from the execution context stack.
             1. Resume the context that is now on the top of the execution context stack as the running execution context.
+            1. If _result_ is an abrupt completion,
+              1. Set _module_.[[Status]] to `"errored"`.
+              1. Set _module_.[[ErrorCompletion]] to _result_.
+            1. Otherwise, set _module_.[[Status]] to `"evaluated"`.
```

Comment:
Setting the module status to "evaluated" at this point will cause an infinite loop if we have a cycle in the module graph.  We either need to set this earlier or set an 'evaluating' status while a module is evaluating.